composer.json

```diff
@@ -12,14 +12,14 @@
   "require": {
     "momentohq/client-sdk-php": "0.2.1"
   },
+  "require-dev": {
+    "orchestra/testbench": "^7.11"
+  },
   "autoload": {
     "psr-4": {
       "Momento\\Cache\\": "src/"
     }
   },
-  "config": {
-    "preferred-install": "dist"
-  },
```

Comment:
Is there a reason you removed this? In the `composer.json` for the SDK, I added the following:

```
  "config": {
    "platform": {
      "php": "8.0"
    },
```
which tells composer to calculate the dependencies in `composer.json` for PHP 8.0 (our minimum supported version) instead of the version of PHP you happen to be running when you run `composer update`. I'd like us to add that to all of our `composer.json` files so packages don't sneak in that are incompatible for some users.

I'm not too concerned about the `preferred-install` property, and I think "dist" is the default anyhow, but I'd definitely appreciate it if you could add the `platform` property.

Reply:
Yeah, as you mentioned, I noticed that `dist` was the default for `preferred-install` so I removed it.
But yes, I agree that we should be consistent about PHP version. I'll add this configuration to `composer.json`.